### DISCUSSION TITLE
How to get the standings for a session

### DISCUSSION THREAD

**Discussion by despolov** [br-g/openf1#25] [Q&A] [answered]
How to get the standings for a session  e.g. Practice 1 from Bahrain. Im trying to do it with /position?session_key=9465 but this returns all of the positions for a driver for this session. In my case I would like to get the position in which he finished in the session. I tried passing "date" too with value date_end from the session object, but thats the end and no data is returned.

**Comment by connor-rose** [comment]
I came to ask this exact question myself! I tried the exact same methods above and ran into the same limitations. Any advice would be awesome!

**Reply by despolov** [reply]
Hi, at the end I got the results but in a different way, not the most efficient, but I already had the data fetched.
 
I have an app in which I want to see the aggregated lap of a driver for each Practice session, meaning I want the fastest sector of a driver no mather in which lap he set it. So I get the session, get the drivers for session, forEach the drivers and get all of the laps for a driver in that session. With some Js logic I forEach the laps and get the fastest sectors and get the fastest lap too for each driver. 

If you are interested you can check the code here, the func name: `getSinglePracticeStats`
[despolov-f1-stats](https://github.com/despolov/f1-stats/blob/main/src/routes/PracticeStats/PracticeStats.js#L78)

**Reply by KevinCamacho** [reply]
Also did something similar for getting the race standings. Get the full list of position data for a race session, sort it by time descending, take the first entry for each driver I see, that is the last position they occupied in the race. (this doesn't provide a way to know if the result was a DNF)

**Comment by br-g** [comment, marked as answer]
Sorry, standings are not available yet through the API!
